### PR TITLE
CLI Wallet Docs

### PR DESCRIPTION
**Pull request by lthiery**
This PR imports and expands upon the [`helium-wallet-rs` docs](https://github.com/helium/helium-wallet-rs/blob/master/README.md). Some of the common "complicated" commands around wallet creation and managed are covered. Emphasis is put on learning how to use the help menu.

### CODE REVIEW

File: docs/wallets/cli.mdx [hunk 1]
```diff
@@ -0,0 +1,277 @@
+---
+id: cli-wallet
+sidebar_label: CLI
+slug: /wallets/cli-wallet
+hide_title: true
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
```
Comment (kent-williams) [inline docs/wallets/cli.mdx:7]: If you move this under the title, the title should render correctly.

File: docs/wallets/cli.mdx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+Enter the command:
+
+```
+$ helium-wallet create basic --seed
+```
+
+After you press enter, you will be prompted for your seed words.
+
+::: info
```
Comment (kent-williams) [inline docs/wallets/cli.mdx]: Guessing prettier did this

Comment (lthiery) [reply docs/wallets/cli.mdx]: whats wrong with it?

Comment (kent-williams) [reply docs/wallets/cli.mdx]: you need a line between this info wrapper I think, it isn't rendering right now

**Comment by lthiery** [comment]
holding until we update with any new seed updates

**Review by kent-williams — APPROVED** [review]

**Pull request merged**